### DISCUSSION TITLE
Error on exporting forum threads

### DISCUSSION THREAD

**Discussion by nintendraw** [Tyrrrz/DiscordChatExporter#1153] [Q&A] [answered]
(The last similar issue was raised [one year ago](https://github.com/Tyrrrz/DiscordChatExporter/discussions/960) and  remains unanswered.)

I RP with people in a personal server and store threads inside forum channels. When I try to use GUI DCE to export the forum channel itself (which has no messages of its own), I get an error: "Channel is a forum and cannot be exported directly. You need to pull its threads and export them individually." Do I need to give DCE the thread ID, or how do I get around this? Thanks!

**Comment by 96-LB** [comment, marked as answer]
Turn on the setting to show threads:
![image](https://github.com/Tyrrrz/DiscordChatExporter/assets/21089343/878f563d-f3e5-4a0e-9660-52a5f251d273)

Then you'll be able to export the individual threads within the forum channel.

**Reply by nintendraw** [reply]
Oops, so it was obvious all along. Thank you!

**Reply by ProjectCisco** [reply]
this didn't fix's the issue for me

**Reply by Tyrrrz** [reply]
Note that the threads will show at the bottom of the list, separate from the forum channel

**Comment by voxelbustersold** [comment]
How this can be done via CLI?

**Reply by AblazingHeart** [reply]
AFAIK it's `--include-threads All`

**Reply by ntevenhere** [reply]
Why isn't this just the default?

**Reply by ntevenhere** [reply]
The explanation appears to be:
> Note that pulling threads (especially archived threads) can be a very time-consuming process, depending on the guild size. 
--- from [changelog](https://github.com/Tyrrrz/DiscordChatExporter/blob/2.42.5/Changelog.md#user-content-v241-15-sep-2023)

**Comment by reverseimaku** [comment]
I'm two years late but make sure your forums are public as well, or the threads won't show in the GUI.
